ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -2136,7 +2140,8 @@ objects:
         function approve_csrs() {
           while true; do
             if [[ ! -f /tmp/install-complete ]]; then
-              oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve
+              # even if oc get csr fails continue
+              oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve || true
```

Comment:
How was this failing? API down or something along those lines?

Reply:
@sdodson my guess is the API isn't 100% ready.  I was testing with another `oc get csr` above the approve then it would always succeed so I didn't get the output I wanted to know exactly.